packages/theme/src/cli/utilities/theme-environment/proxy.ts

```diff
@@ -254,6 +255,10 @@ function proxyStorefrontRequest(event: H3Event, ctx: DevServerContext) {
       redirect: 'manual',
     },
     async onResponse(event, response) {
+      if (!event.path.startsWith(EXTENSION_CDN_PREFIX) && !event.path.startsWith(VANITY_CDN_PREFIX)) {
```

Comment:
We should only show useful lines so perhaps we should filter this more. Some ideas:
- Filter out any pathname ending with an extension, like `.json`,`.js`, etc.
- Filter out known URLs like `/checkouts/...`
- Perhaps we should only show POST requests from the proxy logic?

Or any combination of the above? 🤔

Reply:
I've added a new function that takes care of this called `shouldLog`, and it is easy enough to add or remove paths as needed. Currently we're back to just showing the important GET/POST requests.